### PR TITLE
feat(unlock-app) - add field description in zod

### PR DESCRIPTION
**Pull request by kalidiagne**
# Description

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: unlock-app/src/unlockTypes.ts [hunk 1]
```diff
@@ -4,54 +4,197 @@
...
+    .number({
+      description: `Set the max number of memberships a user can purchase. Note: By default, checkout doesn't allow fiddling with quantity. You have to set maxRecipients to allow for changing to quantity.`,
+    })
+    .int()
+    .positive()
+    .optional(),
+  superfluid: z
+    .boolean({
+      description:
+        'When set to true, superfluid will be enabled as payment method for the lock.',
```
Comment (kalidiagne) [inline unlock-app/src/unlockTypes.ts:87]: @julien51 have any better description here? when don't have description in docs. also we need to add it there?

File: unlock-app/src/unlockTypes.ts [hunk 2]
```diff
@@ -4,54 +4,197 @@
...
 
-export const MetadataInputSchema = z.object({
-  name: z.string(),
-  defaultValue: z.string().optional(),
-  type: z.enum(['text', 'date', 'color', 'email', 'url']),
-  required: z.boolean(),
-  placeholder: z.string().optional(),
-  public: z.boolean().optional(), // optional, all non-public fields are treated as protected
-})
+export const MetadataInputSchema = z
```
Comment (kalidiagne) [inline unlock-app/src/unlockTypes.ts:7]: @julien51 any suggestions for better description here?

**Review by searchableguy — APPROVED** [review]

**Comment by searchableguy** [comment]
You can also look through https://docs.unlock-protocol.com/tools/checkout/configuration/#building-your-url for descriptions and examples.

File: unlock-app/src/unlockTypes.ts (hunk 1 above)
Comment (julien51) [reply unlock-app/src/unlockTypes.ts:86]: HA! You merged already so I can't update :/

**Pull request merged**